l2/eigen-tutorials/packages/demo-eigencall/scripts/exec.js

```diff
@@ -142,14 +147,51 @@ const sub_cipher_plain = async (contract, cipher, plain) => {
 
   expect(rlp_encoded_return_value).not.equal(RLP.encode(""))
 
-  var cipher_base64 = RLP.decode(rlp_encoded_return_value)
-  var cipher = BASE64.btoa(cipher_base64)
+  var cipher_base64 = RLP.decode(rlp_encoded_return_value).toString()
+  var cipher = Base64.toUint8Array(cipher_base64)
   return cipher
 }
 
+function ecies_encrypt(public_key, num) {
+  // default option
+  const options = {
+    hashName: 'sha512',
+    hashLength: 64,
+    macName: 'sha256',
+    macLength: 32,
+    curveName: 'prime256v1',
+    symmetricCypherName: 'aes-256-gcm',
+    keyFormat: 'uncompressed',
+    s1: null, // optional shared information1
+    s2: null // optional shared information2
+  }
+
+  const cipher = ecies.encrypt(public_key, (new Uint64BE(num)).toBuffer(), options);
+  
+  return cipher
+}
 
 const main = async () => {
-  await eigenLog('Simple eigenCall demo')
+  await arbLog('Simple eigenCall demo')
+  
+  const res = await fetch('http://localhost:3000/store?digest=1', {
```

Comment:
setup the PCKS endpoint like L2RPC by `requireEnvVariables(['DEVNET_PRIVKEY', 'L2RPC'])`